README.md

```diff
@@ -22,3 +22,63 @@ Trello board: https://trello.com/b/vptWzBnE/generate-privacy-statements-portage
 - If you want to use Google Analytics or to send feedback or link emails, you have to set up proper environment variables. Copy the `.env.example` file into `.env` and set the variables appropriately.
 - Next install the third party modules using `npm install`
 - You can now run the app locally! run `npm run dev` and then open a web browser to `localhost:3000`.
+
+# April 2023 Update
+
+## Running it locally
+At the moment, it is still possible to run locally, one way is simply running `npm run dev`
+
+
+As part of the Migration to AWS from Heroku, infrastructure for this was created and is hosted on AWS.
+
+## Requires
+- Terraform (https://www.terraform.io/)
+- Terragrunt (https://terragrunt.gruntwork.io/)
+
+## How is this repository structured?
+
+The Terraform code contained in `aws` is split into several independent modules that all use their own remote Terraform state file.These modules know nothing about Terragrunt and are used by Terragrunt as simple infrastructure definitions.
+
+The directory structure inside `aws` reflects the split into independent modules. For example, `ecr`, contains the lifecycle policy and configuration that creates the ECR that will be used by the `app (lambda)` module.
+
+At the moment, there is only a production environment in the `env` which contains the Terragrunt scripts.
+
+## What is each Terraform module
+
+`aws/app`
+
+Contains the Lambda function module that has read, write and root user access to the Elastic File System (EFS). Also contains lambda function url that is used by the cloudfront module.
+
+`aws/cloudfront`
+Contains:
+- Certificates: used to request a DNS validated certificate, deploy the required validation records and wait for validation to complete.
+- Cloudfront: creates an Cloudfront web distribution and contains HTTP response headers and their values.
+- IAM
+- Route53
+- S3: holds access logs
+- WAF: WAFv2 Web ACL resource
+
+`aws/ecr`
+Contains the Elastic Container Registry (ECR) Repository, that will hold the docker image
+
+`aws/efs`
+Contains resources to allow the storage for and access of the generated privacy statements
+
+`aws/hosted_zone`
+Manages Route53 Hosted Zone
+
+`aws/vpc`
+Contains the VPC module and ingress and egress rules. It allows the lambda function to be able to access the VPC.
+
+
+## Architectural Decision Record (ADR)
```

Comment:
Could you move this into a new file in a new `adr` folder in the repo?  You can give it a name of something like:
```
adr/2023-04-13-lambda-infrastructure.md
```